app/views/pair_requests/_form.html.erb

```diff
@@ -3,10 +3,7 @@
     <div id="form-container" class='flex flex-col space-y-4 xl:space-y-0 xl:max-w-full xl:inline-flex xl:justify-around xl:space-x-8 xl:flex-row'>
       <div class="flex justify-between space-x-4 align-middle">
         <%= form.label :invitee_id, class: "label label-text" %>
-        <%= form.select :invitee_id, 
-                        User.invitee_select_for(current_user), 
-                        {}, 
-                        { class: 'select select-primary' } %>
+        <%= form.collection_select :invitee_id, User.all, :id, :full_name, {}, { class: 'select select-primary' } %>
```

Comment:
Forgot about this form helper when I initially worked on this code. This obsoletes the need for the `invitee_select_for` scope that we were previously using here.

Reply:
Actually just thought of a small bug with this ... `User.all` will include the current user. Which probably isn't a great idea. I'll push up a fix.